src/test/ts/trashTaxing.test.ts

```diff
@@ -0,0 +1,40 @@
+import { baseIngredientsParser } from '../../functions/parsers/baseIngredientsParser';
+import { foodParser } from '../../functions/parsers/foodParser';
+import { commandJSONFileOutput } from '../../functions/utils/commandJSONFileOutput';
+import { totalMoneyThrownToTrash } from '../../functions/utils/totalMoneyThrownToTrash';
+import { IObjectInWarehouse } from '../../Interface/IObjectInWarehouse';
+
+describe('Trash taxing tests', () => {
+    test(`We have 1010 potatoes in warehouse -> it was  cleaned, 1010 was thrown away. We have to tax it.`, () => {
+        //given
+        const trash: IObjectInWarehouse[][] = [[{ name: `potatoes`, quantity: 505 }], [{ name: `potatoes`, quantity: 505 }]];
+        const baseIngredients = baseIngredientsParser('./src/test/csv/baseIngredients.csv');
+        const food = foodParser('./src/test/csv/baseIngredients.csv');
+        const jsonSource = '../../json/allEnabled.json';
+        const json = commandJSONFileOutput(jsonSource);
+        //when
+        const result = totalMoneyThrownToTrash(trash, food, baseIngredients, json.wasteTax);
```

Comment:
Why do we have to load whole `json` file (which we don't see), just to use single value from it?
Cannot you just use GIVEN:
`const wasteTax = X;`?
Wouldnt be it more readable? And quicker? And not dependant on file, which you could possibly change in another test?